HanoiTowers/client/menu/PremiumThanks.qml

```diff
@@ -34,10 +34,20 @@ Dialog {
 
         }
 
+        Label {
+            text: qsTr("If you want to unlock all levels and also get rid of annoying ads just for you, we have the premium version of the Towers of Hanoi!")
```

Comment:
wrong text, because when this message display then user already unlock all level